src/coprocessor/tracker.rs

```diff
@@ -38,12 +44,17 @@ enum TrackerState {
 #[derive(Debug)]
 pub struct Tracker {
     request_begin_at: Instant,
+    scheduled_at: Instant,
```

Comment:
How about storing these `Instant`s in `TrackerState`?

Reply:
Good idea!